### PR TITLE
Always export image to file

### PR DESCRIPTION
**Pull request by jpawlowsk**

### CODE REVIEW

File: bigflow/cli.py [hunk 1]
```diff
@@ -493,23 +468,26 @@ def _cli_deploy_image(args):
...
 
-    deploy_docker_image(build_ver=build_ver,
+    deploy_docker_image(image_tar_path=image_tar_path,
                         auth_method=args.auth_method,
                         docker_repository=docker_repository,
                         vault_endpoint=_resolve_vault_endpoint(args),
                         vault_secret=vault_secret)
 
 
+def _find_image_file():
```
Comment (beczkowb) [inline bigflow/cli.py:480]: In general, we don't use underscores. If you want to say which element from a module is public, use __all__.

File: bigflow/cli.py [hunk 2]
```diff
@@ -493,23 +468,26 @@ def _cli_deploy_image(args):
...
+    files = glob1("image", "*-*.tar")
+    if len(files):
```
Comment (beczkowb) [inline bigflow/cli.py:482]: `if files:`

File: docs/cli.md [hunk 3]
```diff
@@ -459,14 +427,15 @@ bigflow deploy-dags \
 
 #### Deploy Docker image examples
 
-Upload a Docker image from a local repository using `local_account` authentication.
+Upload a Docker image exported to a `.tar` using `local_account` authentication.
```
Comment (beczkowb) [inline docs/cli.md:430]: If you are talking from the deployment perspective, you are importing a `.tar`.

File: docs/cli.md [hunk 4]
```diff
@@ -459,14 +427,15 @@ bigflow deploy-dags \
...
+First file from `{current_dir}/image` dir with name matching pattern `.*-.*\.tar` will be used.
```
Comment (beczkowb) [inline docs/cli.md:431]: The first file from the ... dir with a name matching pattern ...

File: docs/cli.md [hunk 5]
```diff
@@ -480,17 +449,20 @@ bigflow deploy-image \
 
 #### Complete deploy examples
 
-Upload DAG files from `{current_dir}/.dags` dir and a Docker image from a local repository using `local_account` authentication.
+Upload DAG files from `{current_dir}/.dags` dir and a Docker image from `{current_dir}/image` dir using `local_account` authentication.
```
Comment (beczkowb) [inline docs/cli.md:452]: from the `{current_dir}/.dags` dir

File: docs/cli.md [hunk 6]
```diff
@@ -480,17 +449,20 @@ bigflow deploy-image \
...
 Configuration is taken from `{current_dir}/deployment_config.py`:
 
 ```bash
-bigflow deploy --version 1.0 --config dev
+bigflow deploy --config dev
 ```
 
-The same, but configuration is taken from a given file:
+The same, but configuration and docker image are taken from specified files:
```
Comment (beczkowb) [inline docs/cli.md:459]: The same, but a configuration and a docker image are taken from the specified files.

File: docs/project_setup_and_build.md [hunk 7]
```diff
@@ -164,6 +164,14 @@ bigflow project-version
 >>> 0.35.0
 ```
 
+If needed, you can specify an identity file for ssh, used to push the tag to remote repository.
```
Comment (beczkowb) [inline docs/project_setup_and_build.md:167]: used to push a tag to a remote repository

File: test/test_cli.py [hunk 8]
```diff
@@ -491,22 +461,38 @@ def test_should_call_cli_deploy_image_command__with_explicit_deployment_config_f
...
                                                     vault_secret='secrett')
 
         dc_file.unlink()
 
     @mock.patch('bigflow.cli.load_image_from_tar')
     @mock.patch('bigflow.cli.deploy_docker_image')
-    @mock.patch('bigflow.cli.tag_image')
-    def test_should_call_cli_deploy_image_command__when_all_parameters_are_given_by_cli_arguments_and_image_is_loaded_from_tar(self, tag_image, deploy_docker_image_mock, load_image_from_tar_mock):
+    def test_should_call_cli_deploy_image_command__when_all_parameters_are_given_by_cli_arguments_and_image_is_loaded_from_tar(self, deploy_docker_image_mock, load_image_from_tar_mock):
+        #when
```
Comment (beczkowb) [inline test/test_cli.py:473]: formatting

File: docs/cli.md [hunk 9]
```diff
@@ -279,42 +279,10 @@ bigflow build-package
 
 The `build-image` command builds 
 a Docker image with Python, your project's PIP package, and
-all requirements. It has one optional parameter :
-
-* `--export-image-to-file` &mdash; If set, an image is exported to a `tar` file.
-  If empty, an image is loaded to a local Docker repository.
-                             
-For example, build a Docker image to a local registry:
+all requirements. Next, image is exported to a `tar` file in `{current_dir}/image` dir.
```
Comment (ewebed) [inline docs/cli.md:282]: the image is exported... file in the

File: test/test_cli.py [hunk 10]
```diff
@@ -491,22 +461,38 @@ def test_should_call_cli_deploy_image_command__with_explicit_deployment_config_f
...
+        #then
+        deploy_docker_image_mock.assert_called_with(auth_method='service_account',
+                                                    docker_repository='my-docker-repository',
+                                                    image_tar_path='image-0.0.1.tar',
+                                                    vault_endpoint='my-vault-endpoint',
+                                                    vault_secret='secrett')
+
+    @mock.patch('bigflow.cli.deploy_docker_image')
+    def test_should_find_tar_in_image_directory(self, deploy_docker_image_mock):
         #given
```
Comment (beczkowb) [inline test/test_cli.py:491]: spaces after #

**Comment by bartoszwalacik** [comment]
@jpawlowsk 👍

**Pull request merged**